lib/Twitter/Utils/TextUtils.vala

```diff
@@ -65,7 +65,7 @@ namespace Backend.Twitter.Utils.TextUtils {
             var entity         = TextModule ();
             entity.type        = MENTION;
             entity.display     = "@" + obj.get_string_member ("username");
-            entity.target      = obj.get_string_member ("id");
+            entity.target      = obj.has_member ("id") ? obj.get_string_member ("id") : "by/username/" + obj.get_string_member ("username");
```

Comment:
I'm not sure that should stay as it is, because it would create an username based id in the cache.
However, I have no clue at all how exactly two-month-past myself wanted to develop this further as well...

tldr: I have to take a look at the entire mentions system...

Reply:
This was definitely an ugly workaround that was abusing your string interpolation function to make it work 😁 But as I think I said in the commit message, it's required because some situations (bios on Twitter?) don't include an ID.